### ISSUE TITLE
WCAG: Implement alt tags in Help documents

### ISSUE THREAD

**Issue by dan-bowerman** [fgpv-vpgf/fgpv-vpgf#4083]
The help docs in RAMP3 contain no alt tags for any of their images. This issue is to add them.

**Comment by dan-bowerman** [comment]
[fr-CA.md](https://github.com/fgpv-vpgf/fgpv-vpgf/files/10339609/fr-CA.md)

**Comment by dan-bowerman** [comment]
[en-CA.md](https://github.com/fgpv-vpgf/fgpv-vpgf/files/10362696/en-CA.md)

**Comment by james-rae** [comment]
It would appear that whatever we are using to convert the markdown to HTML is not respecting the alt definition. Every image has the text `"null"` set as their alt text (it was like this before as well).

**Comment by james-rae** [comment]
We are using `v0.5.2` of the `marked` library; its current version is `v4.2.5` 🤡 . Might be worth trying to update and see if things magically improve.
